### PR TITLE
Update install-kubeadm.md from EN version

### PR DESCRIPTION
**Pull request by rekcah78**

### CODE REVIEW

**Comment by rekcah78** [comment]
Page EN: https://deploy-preview-25592--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

Page FR: https://deploy-preview-25592--kubernetes-io-master-staging.netlify.app/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -80,21 +95,54 @@ les ports personnalisés que vous utilisez sont également ouverts.
...
+|------------|---------------------------------------|
+| Docker     | `/var/run/docker.sock`                |
+| containerd | `/run/containerd/containerd.sock`     |
+| CRI-O      | `/var/run/crio/crio.sock`             |
+{{< /table >}}
+
+<br />
+Si Docker et containerd sont détectés, Docker est prioritaire. C'est
+nécessaire car Docker 18.09 est livré avec containerd et les deux sont détectables même si vous
+installé Docker.
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:129]:
```suggestion
installez Docker.
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -108,17 +156,17 @@ Vous installerez ces paquets sur toutes vos machines:
...
- une erreur de version se produise, qui pourrait conduire à un comportement inattendu.
- Cependant, une version mineure entre les kubelets et le control plane est pris en charge,
- mais la version de la kubelet ne doit jamais dépasser la version de l'API server. Par exemple,
- les kubelets exécutant la version 1.7.0 devraient être entièrement compatibles avec un API
- server en 1.8.0, mais pas l'inverse.
+Vous devez vous assurer qu'ils correspondent à la version du control plane de Kubernetes que vous souhaitez que kubeadm installe pour vous. Si vous ne le faites pas, vous risquez qu'une
+erreur de version se produise, qui pourrait conduire à un comportement inattendu.
+Cependant, une version mineure entre les kubelets et le control plane est pris en charge,
+mais la version de la kubelet ne doit jamais dépasser la version de l'API server.
+Par exemple, les kubelets exécutant la version 1.7.0 devraient être entièrement compatiblesavec un API server en 1.8.0,
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:163]:
```suggestion
Par exemple, les kubelets exécutant la version 1.7.0 devraient être entièrement compatibles avec un API server en 1.8.0,
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 3]
```diff
@@ -108,17 +156,17 @@ Vous installerez ces paquets sur toutes vos machines:
...
+mais pas l'inverse.
+
+For information about installing `kubectl`, see [Install and set up kubectl](/fr/docs/tasks/tools/install-kubectl/).
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:166]:
```suggestion
For information about installing `kubectl`, see [Installation et configuration kubectl](/fr/docs/tasks/tools/install-kubectl/).
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 4]
```diff
@@ -131,125 +179,142 @@ Pour plus d'informations sur les compatibilités de version, voir:
...
+sudo systemctl enable --now kubelet
 ```
 
   **Note:**
 
   - Mettre SELinux en mode permissif en lançant `setenforce 0` et `sed ... `le désactive efficacement.
-  C'est nécessaire pour permettre aux conteneurs d'accéder au système de fichiers hôte, qui
-  est nécessaire par exemple pour les réseaux de pod.
+  C'est nécessaire pour permettre aux conteneurs d'accéder au système de fichiers hôte, qui est nécessaire par exemple pour les réseaux de pod.
     Vous devez le faire jusqu'à ce que le support de SELinux soit amélioré dans la kubelet.
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:218]:
```suggestion
    Vous devez le faire jusqu'à ce que le support de SELinux soit amélioré dans Kubelet.
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 5]
```diff
@@ -131,125 +179,142 @@ Pour plus d'informations sur les compatibilités de version, voir:
...
 ```bash
-CRICTL_VERSION="v1.11.1"
-mkdir -p /opt/bin
-curl -L "https://github.com/kubernetes-incubator/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-amd64.tar.gz" | tar -C /opt/bin -xz
+DOWNLOAD_DIR=/usr/local/bin
+sudo mkdir -p $DOWNLOAD_DIR
 ```
 
-Installez `kubeadm`, `kubelet`, `kubectl` et ajouter un service systemd `kubelet`:
+Installer crictl (requis pour Kubeadm / Kubelet Container Runtime Interface (CRI))
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:244]:
```suggestion
Installez crictl (requis pour Kubeadm / Kubelet Container Runtime Interface (CRI))
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 6]
```diff
@@ -131,125 +179,142 @@ Pour plus d'informations sur les compatibilités de version, voir:
...
+Avant de démarrer votre cluster, vous devez effectuer des étapes supplémentaires pour configurer un répertoire accessible en écriture.
+Consultez le [Guide de dépannage de Kubeadm](/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#usr-mounted-read-only/) pour savoir comment configurer un répertoire accessible en écriture.
+{{< /note >}}
 {{% /tab %}}
 {{< /tabs >}}
 
 
-La kubelet redémarre maintenant toutes les secondes et quelques, car elle attend dans une boucle
-kubeadm, pour lui dire quoi faire.
+Le kubelet redémarre maintenant toutes les quelques secondes,
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:280]:
```suggestion
Kubelet redémarre maintenant toutes les quelques secondes,
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 7]
```diff
@@ -131,125 +179,142 @@ Pour plus d'informations sur les compatibilités de version, voir:
...
+car il attend dans une boucle de crash que kubeadm lui dise quoi faire.
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:281]:
```suggestion
car il attend les instructions de kubeadm dans une boucle de crash.
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 8]
```diff
@@ -131,125 +179,142 @@ Pour plus d'informations sur les compatibilités de version, voir:
...
 
 ## Configurer le driver de cgroup utilisé par la kubelet sur un nœud master
 
 Lorsque vous utilisez Docker, kubeadm détecte automatiquement le pilote ( driver ) de cgroup pour la kubelet
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:285]:
```suggestion
Lorsque vous utilisez Docker, kubeadm détecte automatiquement le pilote ( driver ) de cgroup pour kubelet
```

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 9]
```diff
@@ -131,125 +179,142 @@ Pour plus d'informations sur les compatibilités de version, voir:
...
 ```
 
-Ce fichier sera utilisé par `kubeadm init` et` kubeadm join` pour sourcer des arguments supplémentaires définis par l'utilisateur pour la kubelet.
+Pour plus de détails, veuillez lire [Utilisation de kubeadm init avec un fichier de configuration](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file).
 
 Veuillez noter que vous devez **seulement** le faire si le driver de cgroupe de votre CRI
 n'est pas `cgroupfs`, car c'est déjà la valeur par défaut dans la kubelet.
 
+{{< note >}}
+Depuis que le paramettre `--cgroup-driver` est obsolète par kubelet, si vous l'avez dans`/var/lib/kubelet/kubeadm-flags.env`
```
Comment (remyleone) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:302]:
```suggestion
Depuis que le paramètre `--cgroup-driver` est obsolète par kubelet, si vous l'avez dans`/var/lib/kubelet/kubeadm-flags.env`
```

**Pull request merged**